specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2024-01-01-preview/workspaceRP.json

```diff
@@ -2729,8 +4143,7 @@
       "type": "object",
       "properties": {
         "serviceResourceId": {
-          "type": "string",
-          "format": "arm-id"
+          "type": "string"
```

Comment:
>"type": "string" [](http://example.com/codeflow?start=10&length=16)

Why remove format: arm-id?

Reply:
code autogenerated this with error, reveted